### PR TITLE
Adding Cloudflare 

### PR DESCRIPTION
**Pull request by avicoder**

### CODE REVIEW

**Comment by AfrazHussain** [comment]
I was surprised to see that an organisation like Cloudflare that promotes security so much doesn't offer SSO to customers even on paid plans.

**Comment by derekcwhite** [comment]
This could potentially be updated to clarify that it applies to a subset of Cloudflare's products and services. Cloudflare Zero Trust, for example, has a plethora of SSO options even at the free-for-fifty-users tier. https://www.cloudflare.com/plans/zero-trust-services/

They should still be on this list, but maybe in a slightly smaller font? 

PS: Cloudflare, plz fix already.

**Comment by AfrazHussain** [comment]
True, but it applies to the main Cloudflare product, i.e. Cloudflare Dashboard, which I believe is what is implied by this entry here. I think it should be clarified to say that this only applies to Cloudflare Dashboard and some other Cloudflare services. More on this on the pricing page linked above, and at this [Cloudflare Dashboard SSO guide](https://developers.cloudflare.com/cloudflare-one/applications/configure-apps/dash-sso-apps/).

**Comment by derekcwhite** [comment]
You are correct and I, too, am baffled that they still don't offer SSO at their cheapest paid tier. FWIW, the Cloudflare Dashboard SSO implementation isn't something many people would enjoy paying for, at least as of a year ago when I last used their Dashboad SSO. Currently on my second a-la-carte enterprise plan and didn't bother even asking for Dashboard SSO this time around.

PS: Cloudflare, plz make usable and also available to at least all paid tiers. Thx!

**Comment by abracadabra2** [comment]
Please Re-Add Cloudflare!

File: _vendors/cloudflare.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+base_pricing: $20 per month
+footnotes: 'Enterprise - Custom'
+name: CloudfFlare
```
Comment (Gonent) [inline _vendors/cloudflare.yaml:4]:
```suggestion
name: Cloudflare
```

File: _vendors/cloudflare.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+percent_increase: >900%
+pricing_note: Quote
+pricing_source: https://www.cloudflare.com/en-gb/plans
```
Comment (Gonent) [inline _vendors/cloudflare.yaml:7]:
```suggestion
pricing_source: https://www.cloudflare.com/plans/
```

File: _vendors/cloudflare.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+sso_pricing: > $200 per month
+updated_at: 2022-05-20
```
Comment (Gonent) [inline _vendors/cloudflare.yaml:9]:
```suggestion
updated_at: 2023-04-20
```

File: _vendors/cloudflare.yaml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+base_pricing: $20 per month
+footnotes: 'Enterprise - Custom'
+name: CloudfFlare
+percent_increase: >900%
+pricing_note: Quote
+pricing_source: https://www.cloudflare.com/en-gb/plans
+sso_pricing: > $200 per month
```
Comment (Gonent) [inline _vendors/cloudflare.yaml:8]:
```suggestion
sso_pricing: "> $200 per month"
```

File: _vendors/cloudflare.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+base_pricing: $20 per month
```
Comment (Gonent) [inline _vendors/cloudflare.yaml:2]:
```suggestion
base_pricing: $0
```

**Comment by MMore** [comment]
Hey @avicoder, @Gonent, @AfrazHussain, @abracadabra2, @avicoder, @derekcwhite, @colinbrislawn, @dev-sda1 and @madhifallah ,

Thank you for your contribution! I just integrated your PR into https://github.com/ssotax/ssotax as this repository was not maintained for a long time. Your changes are in https://github.com/ssotax/ssotax/commit/9fa62dbac75b2339b6dba1a669c5fce603c1229c.

You find the new page under https://ssotax.org/.  I also made a few improvements based on community feedback like showing free SSO providers. Feel free to share your feedback!

**Comment by techdarko** [comment]
Addressed in #433, closing.

**Pull request closed without merging**